Sources/StreamChat/Workers/EventNotificationCenter.swift

```diff
@@ -33,61 +29,27 @@ class EventNotificationCenter: NotificationCenter {
         middlewares.append(middleware)
     }
 
-    func process(_ event: Event) {
-        var shouldScheduleProcessing = false
-        _pendingEvents {
-            shouldScheduleProcessing = $0.isEmpty
-            $0.append(event)
-        }
+    func process(_ events: [Event], post: Bool = true, completion: (() -> Void)? = nil) {
```

Comment:
```suggestion
    func process(_ events: [Event], postNotification: Bool = true, completion: (() -> Void)? = nil) {
```

the naming can be a little bit clearer

Reply:
✅ Fixed. Renamed to `postNotifications` in main method and `postNotification` for helper that takes a single event.